modules/local/deseq2_qc/main.nf

```diff
@@ -24,6 +24,16 @@ process DESEQ2_QC {
     path "size_factors"         , optional:true, emit: size_factors
     path "versions.yml"         , emit: versions
 
+    publish:
+    pdf             >> 'deseq2'
```

Comment:
I guess I wanted the publish definition to be in one place so that it's easier to review. But also, I think the separate `publish:` section will work better with static typed outputs

Reply:
`publish:` section removed from process definitions